apps/src/music/player/SamplePlayerWrapper.ts

```diff
@@ -100,6 +100,22 @@ class SamplePlayerWrapper implements AudioPlayer {
     this.samplePlayer.stopAllSamplesStillToPlay();
   }
 
+  setLoopEnabled(): void {
+    console.log('Not supported');
+  }
+
+  setLoopStart() {
+    console.log('Not supported');
+  }
+
+  setLoopEnd() {
+    console.log('Not supported');
+  }
+
+  jumpToPosition() {
+    console.log('Not supported');
+  }
+
```

Comment:
Would it better to make the messages descriptive so we know what's called? e.g.

```
setLoopEnabled(): void {
  console.log('setLoopEnabled is not supported');
}
```

But I'm not sure if that'd just add more noise to it, so I'll defer. Also, could these be better sent out via `console.error`? Again, no strong preference, just raising an option.

Reply:
Good point, I'll update these. In theory we should never hit these so probably better to make them more descriptive if it does somehow occur.